connector/servicegraphconnector/factory.go

```diff
@@ -30,8 +30,8 @@ var virtualNodeFeatureGate, legacyMetricNamesFeatureGate, legacyLatencyUnitMsFea
 func init() {
 	virtualNodeFeatureGate = featuregate.GlobalRegistry().MustRegister(
 		virtualNodeFeatureGateID,
-		featuregate.StageAlpha,
-		featuregate.WithRegisterDescription("When enabled, when the edge expires, processor checks if it has peer attributes(`db.name, net.sock.peer.addr, net.peer.name, rpc.service, http.url, http.target`), and then aggregate the metrics with virtual node."),
+		featuregate.StageBeta,
+		featuregate.WithRegisterDescription("When enabled, when the edge expires, processor checks if it has peer attributes(`peer.service, db.name, db.system`), and then aggregate the metrics with virtual node."),
```

Comment:
```suggestion
		featuregate.WithRegisterDescription("Enabled by default, when the edge expires, processor checks if it has peer attributes(`peer.service, db.name, db.system`), and then aggregate the metrics with virtual node."),
```

Reply:
Other beta features also start with "When enable" so I wanted to avoid breaking uniformity, but this description could indeed be improved, here is what I came up with:

> When enabled and setting `virtual_node_peer_attributes` is not empty, the connector looks for the presence of these attributes in span to create virtual server nodes.